Beelina.APP/src/app/_guards/access.guard.ts

```diff
@@ -0,0 +1,40 @@
+import { Injectable } from '@angular/core';
+import {
+  Route,
+  UrlSegment,
+  ActivatedRouteSnapshot,
+  RouterStateSnapshot,
+} from '@angular/router';
+
+import { Observable } from 'rxjs';
+import { SidedrawerService } from '../_services/sidedrawer.service';
+import { AuthService } from '../_services/auth.service';
+import { NotificationService } from '../shared/ui/notification/notification.service';
+
+@Injectable()
+export class AccessGuard {
+  constructor(
+    private authService: AuthService,
```

Comment:
Unused angular service. Kindly remove.

Reply:
Well, its actually used when automatically logging out the user if unauthorized to access the page. **NO ACTION**

Reply:
Yeah you are right. Then pull request is now approved!